docs/user/openstack/install_upi.md

```diff
@@ -264,16 +264,33 @@ $ tree
 └── install-config.yaml
 ```
 
-### Fix the Node Subnet
+### Configure the machineNetwork CIDR, apiVIP, and ingressVIP.
+The machineNetwork represents the OpenStack network which will be used to connect all the OpenShift nodes that make up the
```

Comment:
Filenames and attributes should be enclosed with backticks: `machineNetwork`, `os_subnet_range`, `inventory.yaml`, etc.